### PR TITLE
Update NONOTICE and NOTICE

### LINKED ISSUE #839: src and binaries include needs correct LICENSE/NOTICE
- typescript is listed in the NONOTICE file but is a dev Dependency and isn't bundled in the vsix file and so does not need to be mentioned
- org.apache.tika.tika-core-2.7.0.jar is bundled as part of omega-edit. It is licensed under ALv2 so is compatible, but its NOTICE content is
   is missing from build/package/NOTICE

Source of this ticket is approval VOTE email from @stevedlawrence for v1.3.1-rc2.

### PR DESCRIPTION
**Pull request by shanedell**
Update NONOTICE and NOTICE

- Add missing dependcies to NOTICE and NOTICE files.
- Clean up the NOTICE information for Apache Pekko:
  - Everything used to be a single repo but seems they split into multiple now.
- Fix version in package.json to go back to a SNAPSHOT version.

Closes #839

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "apache-daffodil-vscode",
   "displayName": "Apache Daffodil™ Extension for Visual Studio Code",
   "description": "Apache Daffodil™ Extension for Visual Studio Code providing DFDL syntax highlighting, DFDL code completion, DFDL schema debugging, and data editor",
-  "version": "1.3.1",
+  "version": "1.4.0-SNAPSHOT",
```
Comment (scholarsmate) [inline package.json:5]: This is interesting, and I agree with this change.  After we do a release, we should probably do this as the next merged commit, until the next release and so on.

**Review by scholarsmate — APPROVED** [review]
+1, LGTM.

**Review by NolanMatt — APPROVED** [review]
Good changes to get rid of the old 1.3 stuff

**Comment by shanedell** [comment]
@mbeckerle @stevedlawrence @arosien @stricklandrbls @rthomas320 Would one of you mind giving this a review/approval?

**Review by stricklandrbls — APPROVED** [review]
+1

**Pull request merged**